app/views/mailer/mfa_recommendation_announcement.html.erb

```diff
@@ -0,0 +1,128 @@
+<% @title = "Enable multi-factor authentication (MFA) on your RubyGems account" %>
+<% @sub_title = "👋 Hi #{@user.handle}" %>
+
+<!-- Body -->
+<table width="100%" border="0" cellspacing="0" cellpadding="0" bgcolor="#ffffff">
+  <tr>
+    <td class="content-spacing" style="font-size:0pt; line-height:0pt; text-align:left" width="20"></td>
+    <td>
+      <table width="100%" border="0" cellspacing="0" cellpadding="0" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%"><tr><td height="35" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%">&nbsp;</td></tr></table>
+
+      <table width="100%" border="0" cellspacing="0" cellpadding="0" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%"><tr><td height="40" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%">&nbsp;</td></tr></table>
+
+      <div class="h3-1-center" style="color:#1e1e1e; font-family:Georgia, serif; min-width:auto !important; font-size:20px; line-height:26px;">
+        <p>
+          Today, we've announced our security-focused ambitions to the community. 
+          Attacks on the software supply chain are increasing and the second most common attack is account takeovers.
+        </p>
+        <br/>
+
+        <p>
+          On August 15, 2022, we will begin requiring maintainers of packages with more than 180 million downloads to have multi-factor authentication (MFA) enabled.
+          Furthermore, we are officially recommending that maintainers of packages with more than 165 million downloads enable MFA. 
+          This recommendation will be repeated by the gem client. 
```

Comment:
Repeated seems out of place. Perhaps:
> This recommendation will be also be shown by the gem client.

Reply:
Done ✅

Reply:
✅ Done (https://github.com/rubygems/rubygems.org/pull/3089/commits/7e913b797a5559160fa7ce4d3720427eacb42165)